plugins/ComposeAuth/README.md

```diff
@@ -0,0 +1,37 @@
+# Supabase-kt ComposeAuth
+
+Extends GoTrue-kt with composable
+
+# Installation
+
+
+Install plugin in main SupabaseClient. See the [documentation](https://supabase.com/docs/reference/kotlin/initializing) for more information
+```kotlin
+val client = createSupabaseClient(
+    supabaseUrl = "https://id.supabase.co",
+    supabaseKey = "apikey"
+) {
+    //...
+    install(GoTrue){
+        //your config
+    }
+    install(ComposeAuth) {
+        googleLoginConfig = googleNativeLogin(serverClientId = "google-client-id")
+        appleLoginConfig = appleNativeLogin()
```

Comment:
Looking at this syntax we should either provide a method to set the login configs for us:
```kotlin
appleNativeLogin()
googleNativeLogin()
```
Or set them ourselves:
```kotlin
googleLoginConfig = GoogleLoginConfig()
```
but both combined seems unnecessary